sklearn/linear_model/tests/test_sgd.py

```diff
@@ -1560,7 +1560,7 @@ def test_multi_core_gridsearch_and_early_stopping():
         'n_iter_no_change': [5, 10, 50],
     }
 
-    clf = SGDClassifier(tol=1e-2, max_iter=1000, early_stopping=True,
```

Comment:
This needs to be reverted because:

https://github.com/scikit-learn/scikit-learn/blob/1a14920464acd9c16fcc0187f89cbbd738068ebc/sklearn/linear_model/tests/test_sgd.py#L75-L77

Changes the `max_iter=5` for all tests in this file.